helix-core/src/selection.rs

```diff
@@ -761,12 +757,7 @@ pub fn select_on_matches(
     None
 }
 
-// TODO: support to split on capture #N instead of whole match
```

Comment:
Should this TODO be kept?

Reply:
altough captures are kind of slow right now with regex-cursor since they fallback to the pikevm as I haven't implemented the bounded backtrack and onepass dfa engines yet. Its not really a problem for most regexes and reasonably sized files (most of these optimzations were only added in very recent regex versions so these aren't super crucial as we didn't have issues with old regex versions)